crates/transaction-pool/src/validate/eth.rs

```diff
@@ -279,14 +279,18 @@ where
 
         // Drop non-local transactions with a fee lower than the configured fee for acceptance into
         // the pool.
-        if !self.local_transactions_config.is_local(origin, transaction.sender_ref()) &&
-            transaction.is_eip1559() &&
```

Comment:
I think we only need to change this check to `is_dynamic_fee()`

Reply:
Cool you are right